### PR TITLE
change reading of F8_AUTOMATED_UPDATE_ENABLED var to be set as param

### PR DESCRIPTION
**Pull request by MatousJobanek**
https://github.com/openshiftio/saas-openshiftio/pull/1170

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @MatousJobanek for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


Your plugin configuration is stored in the [file](https://github.com/fabric8-services/fabric8-tenant/blob/872f0e87a1f3be3d1d26294c606aa2d2940eaaa2/.ike-prow/test-keeper.yaml).

File: OpenShiftTemplate.yml [hunk 1]
```diff
@@ -184,3 +181,5 @@ parameters:
   value: quay.io/openshiftio/rhel-fabric8-services-fabric8-tenant
 - name: IMAGE_TAG
   value: latest
+- name: f8_automated_update_enabled
```
Comment (nurali-techie) [inline OpenShiftTemplate.yml]: don't you want to have upper case `F8_AUTOMATED_UPDATE_ENABLED` as in case of other parameters.

Comment (MatousJobanek) [reply OpenShiftTemplate.yml]: I followed the same approach as is already used in UI repo: https://github.com/fabric8-ui/fabric8-ui/blob/master/openshift/fabric8-ui.app.yaml#L171 https://github.com/openshiftio/saas-openshiftio/blob/master/dsaas-services/fabric8-ui.yaml#L11

Comment (xcoulon) [reply OpenShiftTemplate.yml]: well, I would have preferred that we remain consistent within this template, hence using `AUTOMATED_UPDATE_ENABLED` (no `f8_` prefix and uppercased) would have been better, IMO.

**Pull request merged**